doc/EN/syntax/02_name.md

```diff
@@ -113,12 +113,12 @@ You can delete an variable by using the `Del` function. All other variables that
 ```python,checker_ignore
 x = 1
 y = 2
-Z = 3
+z = 3
```

Comment:
I think this is better in lower case for uniformity

Reply:
You're correct, since constants cannot be deleted.